snuba/query/snql/parser.py

```diff
@@ -1119,6 +1161,33 @@ def mangle_column_value(exp: Expression) -> Expression:
         query.transform_expressions(mangle_column_value)
 
 
+def validate_identifiers_in_lambda(
```

Comment:
one thing I remembered before you merge. You don' test the recursive case of this function, you just have the [one test](https://github.com/getsentry/snuba/pull/2333/files#diff-4876cb16271701ca5a31e45e8df4ccaccc894b2898686d5184b756d1125dd4a9R20-R24). You may want to do that

Reply:
Thanks for the suggestion, I added a new test and tweaked the code as a result.